themes/_core/js/cart.js

```diff
@@ -4,13 +4,13 @@ import prestashop from 'prestashop';
 $(document).ready(() => {
   prestashop.on('cart updated', function (event) {
     var refreshURL = $('.-js-cart').data('refresh-url');
-    var requestData = {};
+    var requestData = {
+      action: 'refresh'
```

Comment:
Sorry, I've mixed up with this https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/templates/checkout/_partials/cart-detailed.tpl#L1
Yes, it would be great if the two were the same :smile:
Thank you